stellar/README.md

```diff
@@ -79,61 +80,63 @@ node stellar/deploy-contract.js --contractName axelar_operators --wasmPath ../ax
+node stellar/generate-bindings.js --wasmPath /path/to/optimized.wasm --contractId [contract_address] --outputDir ./stellar/bindings/[contract name]
 ```
 
 ## Contract Interaction
 
 Soroban contracts can be interacted directly via the CLI as well. See the help text for individual contract cmds as follows.
 
 ```bash
-soroban contract invoke --network testnet --id [contract-address] --source-account wallet -- --help
+stellar contract invoke --network testnet --id [contract_address] --source-account wallet -- --help
 ```
 
 ### Gateway
 
 To get help on the gateway commands, run:
-`node stellar/gateway.js --help`
+
+```bash
+node stellar/gateway.js --help
+```
 
 #### Call contract
 
 ```bash
 node stellar/gateway.js call-contract [destination_chain] [dstination_address] [payload]
 
 # Example
-node stellar/gateway.js call-contract ethereum 0x4F4495243837681061C4743b74B3eEdf548D56A5 0x1234
+node stellar/gateway.js call-contract avalanche 0x4F4495243837681061C4743b74B3eEdf548D56A5 0x1234
 ```
 
 ### Submit multisig prover proof
 
 Submit a proof constructed on Amplifier to the Stellar gateway contract.
 
 ```bash
-node stellar/gateway.js submit-proof [multisig-session-id]
+node stellar/gateway.js submit-proof [multisig_session_id]
 ```
 
-#### Approve messages
+### Approve messages
 
-A message approval can be submitted to the gateway contract for a test deployment where the wallet is the signer on the gateway. Setting `[destination address]` to `wallet` will use the wallet address as the destination.
+A message approval can be submitted to the gateway contract for a test deployment where the wallet is the signer on the gateway. Setting `[destination_address]` to `wallet` will use the wallet address as the destination.
 
 ```bash
-node stellar/gateway.js approve [source-chain] [message-id] [source-address] [destination-address] [payload]
+node stellar/gateway.js approve [source_chain] [message_id] [source_address] [destination_address] [payload]
 ```
 
-#### Validate messages
+### Validate messages
 
 An approved message can be validated by the gateway contract for a test deployment as follows:
 
 ```bash
-node stellar/gateway.js validate-message [source-chain] [message-id] [source-address] [payload]
+node stellar/gateway.js validate-message [source_chain] [message_id] [source_address] [payload]
 ```
 
-#### Rotate signers
+### Rotate signers
 
 A signer rotation can be submitted to the gateway contract. Use `--currentNonce` to override the default current nonce set for subsequent rotations. Skip `--signers` to rotate to the Amplifier verifier set registered in the prover contract.
 
 ```bash
-node node stellar/gateway.js rotate --newNonce test --signers wallet
-
-node node stellar/gateway.js rotate --newNonce test2 --currentNonce test --signers wallet
+node stellar/gateway.js rotate --newNonce test --signers wallet
+node stellar/gateway.js rotate --newNonce test2 --currentNonce test --signers wallet
```

Comment:
```suggestion
node stellar/gateway.js rotate --new-nonce test2 --current-nonce test --signers wallet
```
mind fixing the naming for these as well?

Reply:
`newNonce` and `currentNonce` options are used in various places. and they're also used as variable names that need to be as Carmel Case. I'd prefer to keep them as they are, let me know your thoughts.
https://github.com/axelarnetwork/axelar-contract-deployments/blob/e5ccf0e9fdbcaddca226302835a603097bdb2e41/stellar/gateway.js#L15
https://github.com/axelarnetwork/axelar-contract-deployments/blob/e5ccf0e9fdbcaddca226302835a603097bdb2e41/sui/gateway.js#L226

+ other cases:
https://github.com/axelarnetwork/axelar-contract-deployments/blob/e5ccf0e9fdbcaddca226302835a603097bdb2e41/evm/governance.js#L495

Reply:
we want to move towards a consistent naming scheme, even if other places are inconsistent atm. The convention is that the JS variable should be camel case, but the CLI flags/naming should be kebab case. So can you do `--contract-name <contractName>` for example, or another way to define it with the cli library?

Reply:
it worked nicely! done.